addons/web_editor/static/src/js/rte.js

```diff
@@ -533,9 +533,8 @@ var RTE = Widget.extend({
             }, 150); // setTimeout to remove flickering when change to editable zone (re-create an editor)
             this.$last = null;
         }
-        if ($editable.length && (!this.$last || this.$last[0] !== $editable[0]) &&
-                ($target.closest('[contenteditable]').attr('contenteditable') || "").toLowerCase() !== 'false') {
-
+        if (($editable.length && ($target.closest('[contenteditable]').attr('contenteditable') || "").toLowerCase() !== 'false' || dom.isImg($target[0]))
+            && (!this.$last || this.$last[0] !== $editable[0])) {
```

Comment:
Thanks for the change but after trying to better understand the fix, I think this is not the right way. There may be `<img/>` elements with a `contenteditable="false"` parent which are meant to not be editable. I think the fix should be in the gallery snippet option. In the option, you can see this:
```javascript
    start: function () {
        this._super();
        
        ....

        this.$target.attr("contentEditable", false);

        ...
    },
```
I think the fix should there. We set the whole snippet to `contenteditable="false"`, I think that adding `contenteditable="true"` on the inside images should be enough to fix the problem (see `images_add`) ?